### ISSUE TITLE
Train model: unet-resnet18-imagenet-scse + L1

### ISSUE THREAD

**Issue by joanpuigsanz** [aidl-upc-2024-clothes-extractor/clothes-extractor#20]
Epochs: 200
Scheduler: NO

python entrypoint.py \
    --dataset-device="cpu" \
    --workers=8 \
    --dataloader-pin-memory \
    --num-epochs=200 \
    --max-models-to-keep=5 \
    --model-name=unet-resnet18-imagenet-scse
